pcsx2-qt/Settings/AudioSettingsWidget.cpp

```diff
@@ -114,12 +114,18 @@ AudioSettingsWidget::AudioSettingsWidget(SettingsDialog* dialog, QWidget* parent
 	dialog->registerWidgetHelp(m_ui.syncMode, tr("Synchronization"), tr("TimeStretch (Recommended)"),
 		tr("When running outside of 100% speed, adjusts the tempo on audio instead of dropping frames. Produces much nicer fast-forward/slowdown audio."));
 
-	dialog->registerWidgetHelp(m_ui.expansionMode, tr("Expansion"), tr("Stereo (None, Default)"), tr(""));
+	dialog->registerWidgetHelp(m_ui.expansionMode, tr("Expansion"), tr("Stereo (None, Default)"), 
+		tr("Determines the speaker configuration to apply."));
 
 	//: Cubeb is an audio engine name. Leave as-is.
-	dialog->registerWidgetHelp(m_ui.outputModule, tr("Output Module"), tr("Cubeb (Cross-platform)"), tr(""));
+	dialog->registerWidgetHelp(m_ui.outputModule, tr("Output Module"), tr("Cubeb (Cross-platform)"), 
+		tr("Determines which SPU2 Output Module to use."));
 
-	dialog->registerWidgetHelp(m_ui.backend, tr("Output Backend"), tr("Default"), tr(""));
+	dialog->registerWidgetHelp(m_ui.backend, tr("Output Backend"), tr("Default"), 
+		tr("Determines which System Output Backend to use."));
```

Comment:
I'd say something along the lines of `When the output module supports multiple audio backends, determines the API used to output audio to the system.`

Reply:
Is this `output module` referring to the system's output module?

Reply:
Nope. Output module is Cubeb/XAudio/Null.

Reply:
I guess it's easier to understand as the Audio API, just like how the renderers like Vulkan/DX/OpenGL/... are graphics APIs (in my opinion).

Reply:
Not exactly. While XAudio is the same, Cubeb has multiple backends of its own, corresponding to the host APIs. Hence why I used the term "library" above, rather than "API".

Reply:
I will stick to Library then